src/System.Windows.Forms/src/System/Windows/Forms/ListBox.ItemAccessibleObject.cs

```diff
@@ -19,14 +19,14 @@ internal class ListBoxItemAccessibleObject : AccessibleObject
             private readonly ItemArray.Entry _itemEntry;
             private readonly ListBoxAccessibleObject _owningAccessibleObject;
             private readonly ListBox _owningListBox;
-            private readonly IAccessible? _systemIAccessible;
+
+            private IAccessible? _systemIAccessible => _owningAccessibleObject.GetSystemIAccessibleInternal();
```

Comment:
[naming]
```suggestion
            private IAccessible? SystemIAccessible => _owningAccessibleObject.GetSystemIAccessibleInternal();
```

Properties must be placed under the .ctor.